src/components/image/image.test.tsx

```diff
@@ -0,0 +1,127 @@
+import React from "react";
+import { render, screen } from "@testing-library/react";
+import {
+  testStyledSystemMargin,
+  testStyledSystemLayout,
+  testStyledSystemBackground,
+  testStyledSystemPadding,
+} from "../../__spec_helper__/test-utils";
+import Image from "./image.component";
+
+testStyledSystemMargin((props) => <Image {...props} />);
+testStyledSystemLayout((props) => <Image {...props} />);
+testStyledSystemBackground((props) => <Image {...props} />);
+testStyledSystemPadding((props) => <Image {...props} />);
+
+it("renders", () => {
+  render(<Image data-role="image" />);
+
+  expect(screen.getByTestId("image")).toBeInTheDocument();
```

Comment:
```suggestion
  expect(screen.getByTestId("image")).toBeVisible();
```

**suggestion**: I think its better to use `toBeVisible` as it also asserts it visible to the user, same for the rest of the tests.